style.css

```diff
@@ -0,0 +1,223 @@
+  flex-direction: column;
+}
+
+.game-level {
+  position: relative;
+  outline: none;
+  background: none;
+  color: #fff;
+  font: normal bold 50px/ 59px 'Roboto', sans-serif;
+  align-self: flex-start;
+  margin-left: 6%;
+  cursor: pointer;
+  margin-bottom: 1.8%;
+}
+
+.game-level:hover {
+  -webkit-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  transform: scale(1.1);
+}
+
+#hard {
+  margin-bottom: 0;
+}
+
+/* квадратик */
+
+.active-level::before {
+  background: url(img/square.png) no-repeat;
+  background-size: cover;
+  content: "";
+  width: 25px;
+  height: 25px;
+  position: absolute;
+  top: 27%;
+  right: 110.4%;
+}
+
+.begin {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-gap: .5%;
+}
+
+.begin__button {
+  background: #d62b2e;
+  border-radius: 7px;
+  width: 54.8%;
+  height: 27.8%;
+  color: #fff;
+  font: normal bold 32px/ 37px 'Roboto', sans-serif;
+  margin: 17.6% 0 0 4.5%;
+  outline: none;
+  cursor: pointer;
+}
+
+.begin__button:hover {
+  background: #e79fa0;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 20% 20% 20%;
+}
+
+.card img {
+  border-radius: 4%;
+}
+
+.card__foto {
+  grid-column: 2/3;
+  transform: translate(31px, 19px) rotate(12deg);
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, .7);
+}
+
+.card__pic {
+  z-index: 1;
+  transform: translate(82px, 37px) rotate(39deg);
+  box-shadow: -2px 0 1px rgba(0, 0, 0, .3), 0 4px 4px rgba(0, 0, 0, .25);
+}
+
+.hidden {
+  display: none;
+}
+
+/* Медиазапросы */
+
+@media (max-width: 768px) {
+  .main {
+    max-width: 85%;
+    min-height: 815px;
+  }
+
+  .level {
+    margin-bottom: 8.8%;
+  }
+
+  .level__choice {
+    font: normal bold 54px/ 60px 'Roboto', sans-serif;
+    padding: 12% 0 4% 2.4%;
+  }
+
+  .level__difficulty {
+    font: normal bold 15px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 46px/ 59px 'Roboto', sans-serif;
+  }
+
+  .begin {
+    display: flex;
+  }
+
+  .begin__button {
+    font: normal bold 28px/ 30px 'Roboto', sans-serif;
+    width: 302px;
+    height: 92px;
+    margin: 14% 0 0 4.5%;
+  }
+
+  .card {
+    display: none;
+  }
+}
+
+@media (max-width: 375px) {
+  .main {
+    min-height: 700px;
+  }
+
+  .level {
+    margin-bottom: 12.8%;
+  }
+
+  .level__choice {
+    font: normal bold 44px/ 55px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 12px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 36px/ 59px 'Roboto', sans-serif;
+  }
+
+  .begin__button {
+    font: normal bold 22px/ 30px 'Roboto', sans-serif;
+    width: 252px;
+    height: 72px;
+  }
+}
+
+@media (max-width: 320px) {
+  .main {
+    min-height: 570px;
+  }
+
+  .level {
+    margin-bottom: 7.8%;
+  }
+
+  .level__choice {
+    font: normal bold 36px/ 45px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 10px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 32px/ 59px 'Roboto', sans-serif;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5876445)